lib/flow.js

```diff
@@ -45,6 +46,11 @@ let reportDir = function reportDir(cb) {
   let tsDirName = moment().format("MM-DD-YYYY/HH-mm-ss");
   let fullReportPath = `${reportOutput}/${tsDirName}`;
   this.config.set('output:reports', fullReportPath);
+  this.emitter.on('master:end', (context, summary) => {
+    // fixme when do the date directories get created here? (they always failed for me)
```

Comment:
i think this can be moved into the specific reporter customization sections (xunit/mochawesome) so they aren't even an issue unless you're using those reporters. see: `reporter.js`

Reply:
^ that's on me. i'll add that to the `feature.events` branch